events_webhook/src/main/java/com/adobe/aio/event/webhook/service/PubKeyService.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2017 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+package com.adobe.aio.event.webhook.service;
+
+public interface PubKeyService {
+  String getPubKeyFromCDN(String pubKeyPath);
```

Comment:
It would be nicer to have an more strongly type API here
wdyt we make it `PublicKey getAioEventsPublicKey(String pubkeyPath)`  instead ?

Reply:
I also wonder if you should not remove the use of Feign for this simple cdn download ...
using the simple https://docs.oracle.com/en/java/javase/11/docs/api/java.net.http/java/net/http/HttpClient.html 
could be nicer to foster the use of this library by our customers

Reply:
> 

I wanted to be consistent with the other `aio-lib-java` modules using Feign. Also, don't think using feign would discourage them from using this library.. we can revisit later if such concern arises by users..